src/celeritas/io/ImportPhysicsTable.hh

```diff
@@ -65,10 +67,15 @@ enum class ImportUnits
  */
 struct ImportPhysicsTable
 {
-    ImportTableType table_type;
+    ImportTableType table_type{ImportTableType::size_};
```

Comment:
Why are we initializing this as a `size_`?

Reply:
Since `size_` is always a placeholder meaning "past the last valid enumeration value" I often use it as a sentinel value for "unassigned", since most `switch` statements, `to_cstring` etc. will raise an error when that value is hit.

You do make me think that the `::none` models should be `::other` if they're valid but just not one of the enumerated types (which for now we can't support)? Maybe just delete those entries entirely and change to `::size_`? But that will be a breaking change because it'll invalidate our ROOT files by changing the enumeration...

Reply:
Ohh, good point about returning an error. I like having `other` for valid but not implemented and `size_` as past the last valid enum as a way of saying that we hit an error.
We can defer that to a future PR if you don't want to update the files just yet. Which makes sense, since that will have to be done anyway when the MSC import bit comes into play...